cpp/bindings/bindings.cpp

```diff
@@ -25,45 +29,5 @@ nparr toNumpyArray(const toppra::Matrices& ret) {
   return x;
 }
 
-PyPiecewisePolyPath::PyPiecewisePolyPath(const Matrices& coefficients,
```

Comment:
A few line above this line, at
          for (size_t k = 0; j < x.shape()[2]; j++) x.mutable_at(i, j, k) = ret[i](j, k);

you have a bug in the indices inside the *for* declaration.

Reply:
Thanks!